### PR TITLE
Update to version 7.0.1

### LINKED ISSUE #45: Upgrade binding to support revenuecat sdk 7.0.1
Upgrade binding to support revenuecat sdk 7.1.0

### PR DESCRIPTION
**Pull request by albilaga**
- Update to version 7.0.1
- For Tonestro.Droid.Kotlin.Parcelize.Runtime you can check the source code in here https://github.com/albilaga/Tonestro.Droid.Kotlin.Parcelize.Runtime

Fixes #45

### CODE REVIEW

**Comment by albilaga** [comment]
any idea what causing the error @thisisthekap ?

File: Xamarin.RevenueCat.Android/Xamarin.RevenueCat.Android.csproj [hunk 1]
```diff
@@ -50,6 +50,7 @@
         <Reference Include="Mono.Android" />
         <Reference Include="System" />
         <Reference Include="System.Core" />
+        <Reference Include="System.Core" />
```
Comment (thisisthekap) [inline Xamarin.RevenueCat.Android/Xamarin.RevenueCat.Android.csproj]: duplicate?

**Review by thisisthekap — CHANGES_REQUESTED** [review]

File: Xamarin.RevenueCat.Android/Xamarin.RevenueCat.Android.csproj (hunk 1 above)
Comment (albilaga) [reply Xamarin.RevenueCat.Android/Xamarin.RevenueCat.Android.csproj]: removed

**Review by thisisthekap — APPROVED** [review]

**Pull request merged**